File: Domain/CategoryService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain
{
    public class CategoryService
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public Guid SectorId { get; set; }
        public Sector Sector { get; set; }
        public ICollection<Service> Services { get; set; }

    }
}

File: ScheduledServicesCenterUI/src/app/category.service/category.service.ts
import { Guid } from 'guid-typescript';
import { Sector } from '../sector/sector';
import { Service } from '../service/service';

export interface CategoryService {
  Id: Guid;
  Name: string;
  UrlBackgroundImage:string;
  IsDelete: boolean; 
  SectorId: Guid;
  Sector: Sector;
  Services: Service[];
  /**
   *
   */
}
